viewer/packages/cad-geometry-loaders/src/CadModelUpdateHandler.ts

```diff
@@ -69,14 +79,20 @@ export class CadModelUpdateHandler {
       combineLatest([
         this._loadingHintsSubject.pipe(startWith({} as CadLoadingHints)),
         this._budgetSubject.pipe(startWith(this._budget))
-      ]).pipe(map(makeSettingsInput)),
-      this._prioritizedLoadingHintsSubject.pipe(startWith(undefined as void)),
-      combineLatest([
-        this._cameraSubject.pipe(auditTime(500)),
-        this._cameraSubject.pipe(auditTime(250), emissionLastMillis(600))
-      ]).pipe(map(makeCameraInput)),
-      combineLatest([this._clippingPlaneSubject.pipe(startWith([]))]).pipe(map(makeClippingInput)),
-      this.loadingModelObservable()
+      ]).pipe(map(makeSettingsInput), auditTime(250)),
+      this._prioritizedLoadingHintsSubject.pipe(startWith(undefined as void), auditTime(250)),
+      this._cameraSubject.pipe(
+        auditTime(500),
+        filter((input: CameraInput) => input.cameraInMotion),
+        mergeWith(
+          this._cameraSubject.pipe(
+            distinctUntilKeyChanged('cameraInMotion'),
+            filter((input: CameraInput) => !input.cameraInMotion)
+          )
+        )
+      ),
+      combineLatest([this._clippingPlaneSubject.pipe(startWith([]))]).pipe(map(makeClippingInput), auditTime(250)),
+      this.loadingModelObservable().pipe(auditTime(250))
```

Comment:
Can you give a small TLDR on what the implications are for this change?

Reply:
Sure, and I'll preface it by saying I'm no RxJS wizard, so there's probably a better way to do this. 

Two logical changes have happened in more or less the same place:
1. The "global" 250ms auditTime (old line 109) has been removed, and moved into the different observables 'triggers' above (which deals with settings, loading hint, clipping planes, models). The reason for this is that I didn't want the extra delay when the trigger was the camera.
2. The logic for the 'camera trigger' is altered. Previously, it was a combination of every 500ms, and every 250ms check whether it's more than 600ms since the last camera update (at least that's how I read it..). Now, it emits every 500ms if the camera is moving, as well as the first frame of it not moving. The `pipe(auditTime(500),  filter((input: CameraInput) => input.cameraInMotion))` fires every 500ms when the camera is moving, and the `.pipe(distinctUntilKeyChanged('cameraInMotion'), filter((input: CameraInput) => !input.cameraInMotion))` emits when once when the `cameraInMotion` property changes to false.

As before, `SectorLoader.loadSectors` will not do anything if continous streaming is disabled and the camera is moving.